drivers/serial/uart_stm32.c

```diff
@@ -1695,6 +1695,16 @@ static int uart_stm32_init(const struct device *dev)
 	}
 #endif
 
+	if (config->de_enable && IS_UART_DRIVER_ENABLE_INSTANCE(config->usart)) {
+		LL_USART_EnableDEMode(config->usart);
+		LL_USART_SetDEAssertionTime(config->usart, config->de_assert_time);
+		LL_USART_SetDEDeassertionTime(config->usart, config->de_deassert_time);
```

Comment:
Are those functions compatibles with all STM32 series ?

Reply:
Looks like all except F1xx F2xx F4xx L1xx. Sorry, forgot to check. I can wrap it in a `#ifdef USART_CR3_DEM` (The bit for enabling the driver enable) which is consistent across all compatible series.